src/main/java/com/mongodb/hibernate/jdbc/MongoPreparedStatement.java

```diff
@@ -280,6 +280,11 @@ public void setNull(int parameterIndex, int sqlType, @Nullable String typeName)
         throw new NotYetImplementedException("To be implemented during Array / Struct tickets");
     }
 
+    @Override
+    public boolean isWrapperFor(Class<?> iface) {
```

Comment:
The changes are now smeared across https://github.com/mongodb/mongo-hibernate/pull/24 (this PR) and https://github.com/mongodb/mongo-hibernate/pull/22. Given that this PR is definitely getting merged into `main` before the other one, would you please agree to do all the relevant changes here?

- override `isWrapperFor` in `MongoConnection` and `MongoStatement`;
- once the above is done, we no longer need to override `isWrapperFor` in `MongoPreparedStatement`, ad it extends `MongoStatement`.

Reply:
I definitively hope this PR be merged first for this one blocks everything and is more vital for project progress.

Great point! I did as you suggested.